configure.ac

```diff
@@ -982,6 +982,8 @@ if test "$enable_delayacct" = yes; then
   AC_DEFINE([HAVE_DELAYACCT], [1], [Define if delay accounting support should be enabled.])
   htop_config_cflags=`$PKG_CONFIG --cflags libnl-3.0 2>/dev/null`
   AM_CFLAGS="$AM_CFLAGS $htop_config_cflags"
+  libdir_libnl=`$PKG_CONFIG --variable=libdir libnl-3.0 2>/dev/null`
+  AC_DEFINE_UNQUOTED([LIBDIR_LIBNL], ["$libdir_libnl"], [Path to the libnl-3 libraries])
```

Comment:
Um. A small question (maybe this is bike shedding).
Which name for the shell variable / C macro is better, `LIBNL_LIBDIR` or `LIBDIR_LIBNL`?

I would vote for `LIBNL_LIBDIR`.

Reply:
Other places like for hwloc use `FOO_LIBS`, thus `LIBNL_LIBDIR` would be reasonable.